submissions/lukmanokunade/Database

```diff
@@ -0,0 +1 @@
+https://docs.google.com/document/d/e/2PACX-1vSDyn-sC1xHV9pKw5sfXSDSCjRBbPK1UVmqUhkZ0fELjBiIij7TpHnxeKG8KSf5palTGTY67Y8nkIBg/pub
```

Comment:
link is not opening @lokunade. please check

Reply:
Pls, check again. I allowed the link to be viewed initially.